terraform/00_state.tf

```diff
@@ -16,22 +16,28 @@
 # backend which stores everything in a local directory. The config you see below
 # is functionally equivalent to the defaults.
 
-terraform {
-  backend "local" {
-    path = "terraform.tfstate"
-  }
-}
+#terraform {
+#  backend "local" {
+#    path = "terraform.tfstate"
+#  }
+#}
```

Comment:
Don't leave code remarks in your changelists

Reply:
Ok.